File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/Person.java
package com.kinvey.androidTest.model;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;

import java.util.List;

public class Person extends GenericJson{

    public static final String COLLECTION = "Persons";
    public static final String DELTA_SET_COLLECTION = "QuerySyncCollection";
    public static final String DELTA_SET_OFF_COLLECTION = "DeltaSetNotEnabled";
    public static final String COLLECTION_WITH_EXCEPTION = "CollectionWithException";

    public static final String LONG_NAME = "LoremIpsumissimplydummytextoftheprintingandtypesettingindustry";

    public Person() {
    }

    public Person(String username) {
        this.username = username;
    }

    @Key
    protected String age;

    @Key("_id")
    protected String id;

    @Key
    private float height;

    @Key
    private Author author;

    @Key
    private long weight ;

    @Key
    private Integer carNumber ;

    @Key("username")
    protected String username;

    public String getAge() {
        return age;
    }

    public void setAge(String age) {
        this.age = age;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public float getHeight() {
        return height;
    }

    public void setHeight(float height) {
        this.height = height;
    }

    public long getWeight() {
        return weight;
    }

    public void setWeight(long weight) {
        this.weight = weight;
    }

    public Integer getCarNumber() {
        return carNumber;
    }

    public void setCarNumber(Integer carNumber) {
        this.carNumber = carNumber;
    }

    public Author getAuthor() {
        return author;
    }

    public void setAuthor(Author author) {
        this.author = author;
    }
}

File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/Person.kt
package com.kinvey.androidTest.model

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key

open class Person() : GenericJson() {

    constructor(username: String?): this() {
        this.username = username
    }

    constructor(id: String?, username: String?): this() {
        this.id = id
        this.username = username
    }

    @Key
    var age: String? = null
    @Key("_id")
    var id: String? = null
    @Key
    var height = 0f
    @Key
    var author: Author? = null
    @Key
    var weight: Long = 0
    @Key
    var intVal = 0
    @Key
    var carNumber: Int? = null
    @Key("username")
    var username: String? = null
    @Key("_geoloc")
    var geoloc: String? = null

    companion object {
        const val COLLECTION = "Persons"
        const val DELTA_SET_COLLECTION = "QuerySyncCollection"
        const val TEST_COLLECTION = "TestCollection"
        const val USER_STORE = "UserStore"
        const val DELTA_SET_OFF_COLLECTION = "DeltaSetNotEnabled"
        const val COLLECTION_WITH_EXCEPTION = "CollectionWithException"
        const val USERNAME_KEY = "username"
        const val LONG_NAME = "LoremIpsumissimplydummytextoftheprintingandtypesettingindustry"
    }
}